### PR TITLE
Merge 12.7.0.1 into trunk

### PR DESCRIPTION
**Pull request by spencertransier**
This PR merges the changes from the `12.7.0.1` beta build into `trunk`. Includes: 
- Version bump
- Updated release note translations
- Updated app translations
- Beta fix: https://github.com/woocommerce/woocommerce-ios/pull/9125
- Beta fix: https://github.com/woocommerce/woocommerce-ios/pull/9150

### CODE REVIEW

**Comment by spencertransier** [comment]
@itsmeichigo could you help me with the merge conflicts on this PR? I usually don't try to fix them myself unless it's **very** self-explanatory so that I don't break anything 😄

**Comment by itsmeichigo** [comment]
@spencertransier I removed the changes that caused the conflicts - will raise a PR in `trunk` for similar changes with the new update.

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -12,6 +12,8 @@
 - [*] Stats: Fixed a crash when order stats use a date and time matching the start of Daylight Saving Time. [https://github.com/woocommerce/woocommerce-ios/pull/9083]
 - [*] Fix: Dismiss Take Payment popup after sharing the payment link to another app. [https://github.com/woocommerce/woocommerce-ios/pull/9042]
 - [*] Site credential login: Catch invalid cookie nonce [https://github.com/woocommerce/woocommerce-ios/pull/9102]
+- [*] Better error messages for site credential login failures [https://github.com/woocommerce/woocommerce-ios/pull/9125]
+- [Internal] New Zendesk tag for site credential login errors [https://github.com/woocommerce/woocommerce-ios/pull/9150]
```
Comment (mokagio) [inline RELEASE-NOTES.txt:16]: @spencertransier @itsmeichigo given the merge conflict resolution commits that removed the Zendesk tag, should this entry be removed as well?

**Review by mokagio — APPROVED** [review]
- [x] Updates to the localization files (`.strings` and Fastlane metadata) automatically pulled for the new translations that have already been approved in GlotPress
- [x] Version update in `.xcconfig`
- [x] Diffs from the PRs that made it into this beta

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (itsmeichigo) [reply RELEASE-NOTES.txt:16]: The feature was removed from this branch, but it was merged and built for 12.7.0.1 before, so the release note should still be valid for 12.7 right?

**Pull request merged**